src/data/finset/basic.lean

```diff
@@ -2262,6 +2262,8 @@ theorem card_def (s : finset α) : s.card = s.1.card := rfl
 theorem card_le_of_subset {s t : finset α} : s ⊆ t → card s ≤ card t :=
 multiset.card_le_of_le ∘ val_le_iff.mpr
 
+@[mono] lemma card_mono (α : Type) : monotone (@finset.card α) := by apply card_le_of_subset
```

Comment:
```suggestion
@[mono] lemma card_mono (α : Type*) : monotone (@finset.card α) := by apply card_le_of_subset
```
otherwise this lemma is almost useless!

Is there a reason you make `α` explicit here but not in the multiset lemma? I'd probably lean towards it being implicit.

Reply:
\alpha is already an implicit variable in the environment, I think it shouldn't be necessary at all. Also no need for `finset.`.